modules/asset-manager-storage-aws/src/main/java/org/opencastproject/assetmanager/aws/s3/AwsS3AssetStore.java

```diff
@@ -544,12 +572,24 @@ private void restoreGlacierObject(String objectName, Integer objectRestorePeriod
     }
   }
 
+  private String getBucketName(String orgId) {
```

Comment:
This needs release notes.  Users with existing S3 configs *must* ensure that the default bucket is still available (either as the default, or as an org bucket) otherwise the data won't be accessible since this method won't return the right bucket name